### DISCUSSION TITLE
Which U.S. presidents are depicted on Mount Rushmore?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#690] [Galaxy Trivia] [answered]
*listing from right to left*

**Comment by SANKARAMDAS** [comment, marked as answer]
Abraham Lincoln (1809-1865).
Theodore Roosevelt (1858-1919)
 Thomas Jefferson (1743-1826);
George Washington (1732-1799);

**Comment by peterbaccaro** [comment]
From right to left the US presidents are:

**Abraham Lincoln**, 16th President of the United States.

**Theodore Roosevelt**, 26th President of the United States. Born 1858, died 1919.

**Thomas Jefferson**, Third President of the United States. Born 1743, died 1826.

**George Washington**, First President of the United States. Born 1732, died 1799.
